.github/workflows/fedora-38.yml

```diff
@@ -552,3 +552,108 @@ jobs:
           path: |
             /tmp/artifacts/*.json
             /tmp/artifacts/*.log
+
+  pre-iot-simplified-f38:
+    needs: pr-info
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: Tests / iot-simplified-f38
+
+    steps:
+      - name: Get Current Job Log URL
+        id: job-log-url
+        run: |
+          JOB_HTML_URL=$(curl --get -Ss -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/osbuild/rhel-edge-ci/actions/runs/${GITHUB_RUN_ID}/jobs?per_page=30" | jq -r --arg job_name "pre-iot-simplified-f38" '.jobs | map(select(.name == "pre-iot-simplified-f38")) | .[0].html_url')
+          echo "html_url=$JOB_HTML_URL" >> $GITHUB_OUTPUT
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/osbuild/osbuild-composer/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Fedora 38 - Runner has been deploying...'
+          target_url: '${{ steps.job-log-url.outputs.html_url }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+
+  iot-simplified-f38:
+    needs: [pr-info, pre-iot-simplified-f38]
+    runs-on: [kite, x86_64, gcp, fedora-38, medium]
```

Comment:
Please use openstack vm to run this test to make it more stable.

Reply:
fixed.